NSS-Day-Cohort-53/tabloid-full-stack-always-polite-iguanas
# Bb view all posts

#### Changes Made
1. Added file `PostController.cs` to `Controller` directory.
2. Added file `Post.cs` to `Models`
3. Added files `PostRepository.cs` and `IPostRepository` to `repositories`
4. Added file `Post.js` to `src/components/Posts`
5. Added file `PostList.js` to `src/components/Posts`
6. Added file `postManager.js` to `src/modules`
7. Modified file `Header.js` to include navigation links to posts page for logged in users.
8. Modified file `ApplicationViews.js` to include routes for posts list module
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout bb-ViewAllPosts
    ```
7. Open a new Terminal tab (⌘T) and navigate to the server directory.
8. Test app functionality.
    > When a logged in user clicks on the Posts link they are taken to a list of posts and each entry in the list has author name, title, and content.
9. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
